packages/material-ui/src/index.js

```diff
@@ -254,9 +254,6 @@ export * from './Popover';
 export { default as Popper } from './Popper';
 export * from './Popper';
 
-export { default as Portal } from './Portal';
-export * from './Portal';
-
```

Comment:
Why remove this here?

We still support `import Portal from '@material-ui/core/Portal'`. Why do we drop support for `import { Portal } from '@material-ui/core'`

Reply:
it's not dropped because everything is exported in https://github.com/mui-org/material-ui/blob/fe46b780c11667b967027cc07ebbc5d5b61e60aa/packages/material-ui/src/index.js#L9 but I am missing https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/index.d.ts same export @eps1lon

Reply:
Gotcha, so we just forgot to add the wildcard re-export for `index.d.ts` in https://github.com/mui-org/material-ui/pull/23308?

Reply:
Yes

Reply:
I should really revisit https://github.com/mui-org/material-ui/pull/23166. We break import/exports way too often.